clusters/app.ci/registry-access/telcov10n-ci/telcov10n-ci.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Published Images for telcov10n-ci
+    openshift.io/display-name: telcov10n-ci
+  name: telcov10n-ci
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: telcov10n-ci-image-puller-binding
+  namespace: telcov10n-ci
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:image-puller
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
```

Comment:
Don't we need the LDAP group name here?

Reply:
Not specifically, it means every authenticated user can use the images in this namespace.